GW2EIEvtcParser/EncounterLogic/Strikes/SotO/CosmicObservatory.cs

```diff
@@ -25,16 +25,21 @@ public CosmicObservatory(int triggerID) : base(triggerID)
                 new PlayerDstBuffApplyMechanic(Revealed, "Revealed", new MechanicPlotlySetting(Symbols.Bowtie, Colors.Teal), "Sl.Fst.T", "Soul Feast Target", "Targetted by Soul Feast", 0).UsingChecker((bae, log) => bae.CreditedBy.IsSpecies(TargetID.Dagda)),
-                new PlayerDstBuffApplyMechanic(DagdaSharedDestruction, "Shared Destruction", new MechanicPlotlySetting(Symbols.Circle, Colors.Green), "Shar.Des.T", "Targetted by Shared Destruction (Greens)", "Shared Destruction Target (Green)", 0),
+                new PlayerDstBuffApplyMechanic(DagdaSharedDestruction_MeteorCrash, "Shared Destruction", new MechanicPlotlySetting(Symbols.Circle, Colors.Green), "Shar.Des.T", "Targetted by Shared Destruction (Greens - Meteor Crash)", "Shared Destruction Target (Green - Meteor Crash)", 0),
                 new PlayerDstBuffApplyMechanic(new long [] { TargetOrder1, TargetOrder2, TargetOrder3, TargetOrder4, TargetOrder5 }, "Target Order", new MechanicPlotlySetting(Symbols.Star, Colors.LightOrange), "Targ.Ord.A", "Received Target Order", "Target Order Application", 0),
+                new PlayerDstBuffApplyMechanic(ExtremeVulnerability, "Extreme Vulnerability", new MechanicPlotlySetting(Symbols.X, Colors.DarkRed), "ExtVuln.A", "Applied Extreme Vulnerability", "Extreme Vulnerability Application", 0),
+                new PlayerSrcBuffRemoveSingleFromMechanic(DagdaDemonicAura, "Demonic Aura", new MechanicPlotlySetting(Symbols.Bowtie, Colors.LightBlue), "DemAur.R", "Removed Stacks of Demonic Aura", "Demonic Aura Stacks Removed"),
                 new PlayerSrcSkillMechanic(PurifyingLight, "Purifying Light", new MechanicPlotlySetting(Symbols.Hourglass, Colors.LightBlue), "PurLight.C", "Casted Purifying Light", "Casted Purifying Light", 0),
                 new PlayerSrcHitMechanic(PurifyingLight, "Purifying Light", new MechanicPlotlySetting(Symbols.HourglassOpen, Colors.LightBlue), "PurLight.Soul.C", "Casted Purifying Light (Hit Soul Feast)", "Purifying Light Hit Soul Feast", 0).UsingChecker((ahde, log) => ahde.To.IsSpecies(TrashID.SoulFeast)),
                 new PlayerSrcHitMechanic(PurifyingLight, "Purifying Light", new MechanicPlotlySetting(Symbols.HourglassOpen, Colors.Blue), "PurLight.Dagda.C", "Casted Purifying Light (Hit Dagda)", "Purifying Light Hit Dagda", 0).UsingChecker((ahde, log) => ahde.To.IsSpecies(TargetID.Dagda)).UsingEnable(x => x.FightData.IsCM),
+                new PlayerDstEffectMechanic(EffectGUIDs.CosmicObservatoryDemonicFever, "Demonic Fever", new MechanicPlotlySetting(Symbols.Circle, Colors.LightOrange), "DemFev.T", "Targetted by Demonic Fever (Orange Spread AoEs)", "Demonic Fever Target", 0),
+                new EnemyDstBuffRemoveSingleMechanic(DagdaDemonicAura, "Demonic Aura", new MechanicPlotlySetting(Symbols.BowtieOpen, Colors.LightBlue), "Demonic Aura (Lost)", "Lost stacks of Demonic Aura", "Demonic Aura Stacks Lost").UsingChecker((abre, log) => abre.CreditedBy.IsPlayer),
```

Comment:
"Demonic Aura (Lost)" kinda too long, you should use DemAur.L to be consistent with the  PlayerSrcBuffRemoveSingleFromMechanic version.